ocw_data_parser/ocw_data_parser.py

```diff
@@ -226,6 +227,7 @@ def _compose_page_dict(j):
     def compose_media(self):
         def _compose_media_dict(j):
             return {
+                "order_index": safe_get(j, "actual_file_name").replace(".json", ""),
```

Comment:
Can we keep `actual_file_name` as the key name or `original_file_name`? I think `order_index`  implies that the field will be an integer. Also it will be confusing if a course does not follow the naming convention

Reply:
It does imply that the field will be an integer.  As far as I know, the `0.json` etc. structure is built into Plone and that's the way it's coded to export so we have no reason to expect anything different.  You're right though in that I should do some sanity checking here to make sure it's an integer before setting a value.